src/main/scala/fi/oph/koski/raportit/RaporttiUtils.scala

```diff
@@ -3,8 +3,17 @@ package fi.oph.koski.raportit
 import java.time.LocalDate
 
 import fi.oph.koski.raportointikanta.ROsasuoritusRow
+import fi.oph.koski.schema.Organisaatio.isValidOrganisaatioOid
 
 object RaporttiUtils {
+  def validateOrganisaatioOids(oppilaitosOids: List[String]): List[String] = {
```

Comment:
Voisko toi funktio elää samassa paikassa missä on jo organisaatioOid:in validaatioon liittyvä koodia? Se vois olla helpompi löytää siinä tilanteessa kun haluaa validoida oideja vs. se et sitä ettis yleisestä objectistä mikä voi pitää sisällään mitä vaan

https://github.com/Opetushallitus/koski/blob/master/src/main/scala/fi/oph/koski/organisaatio/OrganisaatioOid.scala#L7

Reply:
Kävi myös mielessä, mut ei jotenkin tuntunut oikealta laittaa sinne metodia joka heittää poikkeuksen niin kuin tää tekee. Nyt kun mietin tätä lisää niin tuohan vois palauttaa Eitherin. Lisäksi organisaatio-oidit pitäisi oikeastaan validoida Servletissä jolloin fiksun http-vastauksen palauttaminen Eitherin perusteella olis helpompaa.

Reply:
itseasiassa sehän tehdään jo
https://github.com/Opetushallitus/koski/blob/master/src/main/scala/fi/oph/koski/raportit/RaportitServlet.scala#L207

Reply:
Jooh näemmä, enpä sit tiedä miksi noissa muissa raporteissa on sit päädytty validoimaan syötetyn oidin lisäksi myös generoidut oidit noin. Mutta en taida koskea niihin sittenkään.